src/UnifiedFFI-Tests/FFIValueHolderTest.class.st

```diff
@@ -151,6 +160,37 @@ FFIValueHolderTest >> testCallIntPointerWithValueHolder [
 	holder value free
 ]
 
+{ #category : 'tests' }
+FFIValueHolderTest >> testCallStructPointerPointerWithValueHolder [
+	| holder result string |
+	
+	holder :=  FFITestStructure newValueHolder.
+	
+	FFICallbackFunctionResolution 
+		registerCallback: (FFICallback 
+			signature: #(int (FFITestStructure **aValue)) 
+			block: [ :aValue |
+				| bytes allocated |
+				bytes := ByteArray streamContents: [ :stream |
+					0 to: 2 do: [ :i | 
+						stream nextPutAll: (FFITestStructure new 
+							long: 42 + i;
+							getHandle) ] ].
+				allocated := ExternalAddress allocate:bytes size.
+				LibC memCopy: bytes to: allocated size: bytes size.
+				aValue pointerAt: 1 put: allocated.
+				3 ])
```

Comment:
Why returning 3 is important? I assume this is the same 3 as in the assertion below. But is 3 important (like, is it the size of the array?)?